### ISSUE TITLE
Missing Functionality: JES Monitoring Script

### ISSUE THREAD

**Issue by cjllanwarne** [broadinstitute/cromwell#1006]

**Comment by ruchim** [comment]
Monitoring.log file is created when monitoring_script WF option is used, but the log file is completely empty. It's possible that the correct outputs aren't copied over to the log file.

**Comment by ruchim** [comment]
Also, from a user perspective, it might be nice to include the monitoring script file path in the workflow metadata.

**Comment by Horneth** [comment]
Depending on what your monitoring script does and how long your command takes to run it's possible that the task finishes before the monitoring script had time to write / flush anything into the monitoring log.
I ran this and got an empty log 
```
task t {
  command {
    echo "hey"
  }

  runtime {
    docker: "ubuntu:latest"
  }
}
workflow w {
  call t
}
```

but this gave me a non-empty one 
```
task t {
  command {
    sleep 5
  }

  runtime {
    docker: "ubuntu:latest"
  }
}
workflow w {
  call t
}
```

**Comment by mcovarr** [comment]
Sounds like this may not be a bug, back to @cjllanwarne for possible rebuttal.

**Comment by cjllanwarne** [comment]
This was set to `ignored` in centaur, so I added the ticket. If it's working already... then we don't need a ticket!

**Comment by mcovarr** [comment]
TL;DR the monitoring script actually does work, but Cromwell and Centaur need a bit of tweaking for this to be testable.

I added a `sleep` to the Centaur test per @Horneth's suggestion and the monitoring log does get written.  However Cromwell never creates a metadata event with the path to the monitoring log.  Also the test needs to know where to expect the monitoring log, which is actually a function of the workflow root which Centaur currently doesn't know.  I'd like to fix this by publishing the workflow root as a metadata event and then making Centaur wise to it.
